test/NuGet.Core.Tests/Microsoft.Build.NuGetSdkResolver.Tests/RestoreRunnerExTests.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Newtonsoft.Json.Linq;
+using NuGet.Commands;
+using NuGet.Commands.Test;
+using NuGet.Configuration;
+using NuGet.LibraryModel;
+using NuGet.Packaging;
+using NuGet.ProjectModel;
+using NuGet.Protocol;
+using NuGet.Protocol.Core.Types;
+using NuGet.Test.Utility;
+using NuGet.Versioning;
+using Xunit;
+
+namespace Microsoft.Build.NuGetSdkResolver.Test
+{
+    public class RestoreRunnerExTests
```

Comment:
What's the "Ex" here, and is it following our naming conventions?

Reply:
The pattern is:

> **ClassName**Tests

The class that is being tested is https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/Microsoft.Build.NuGetSdkResolver/RestoreRunnerEx.cs.